ffi/mupdf.lua

```diff
@@ -108,7 +108,7 @@ function mupdf.openDocument(filename)
 
     setmetatable(mupdf_doc, document_mt)
 
-    if not (mupdf_doc:getPages() > 0) then
+    if mupdf_doc:getPages() <= 0 then
```

Comment:
I probably do prefer this.

Reply:
FWIW, that's the only one that gave me pause (mostly because I don't quite recall what happens on `merror` if `getPages` fail anyway :D).

Reply:
I would assume that it'll fail ungracefully with both writing styles. ^_^